src/main/kotlin/uk/gov/justice/digital/hmpps/notificationsalertsvsip/service/SmsSenderService.kt

```diff
@@ -83,14 +83,13 @@ class SmsSenderService(
   }
 
   private fun getCommonTemplateVars(visit: VisitDto): MutableMap<String, String> {
-    val prisonName = prisonRegisterService.getPrisonName(visit.prisonCode)
-
     val templateVars = mutableMapOf(
-      "prison" to prisonName,
+      "prison" to prisonRegisterService.getPrisonName(visit.prisonCode),
       "time" to dateUtils.getFormattedTime(visit.startTimestamp.toLocalTime()),
       "dayofweek" to dateUtils.getFormattedDayOfWeek(visit.startTimestamp.toLocalDate()),
       "date" to dateUtils.getFormattedDate(visit.startTimestamp.toLocalDate()),
     )
+
     return templateVars
   }
```

Comment:
just a thought should this move to a different service that can be used by both EmailSenderService and SmsSenderService?

Reply:
After discussing, we will leave this pull request as is. Then after merging, we'll work on splitting each email template into their own service.